Docker build is failing

Thanks for creating an issue! Please fill out this form so we can be
sure to have all the information we need, and to minimize back and forth.

* **What are you trying to do?**
build using docker

* **What feature or behavior is this required for?**
to make the jar file.

* **How could we solve this issue? (Not knowing is okay!)**

```docker build -t nexus-repository-conan:0.0.5 .
Sending build context to Docker daemon  509.4kB
Step 1/15 : ARG NEXUS_VERSION=3.14.0
Step 2/15 : FROM maven:3-jdk-8-alpine AS build
 ---> f6ecca2010f4
Step 3/15 : ARG NEXUS_VERSION=3.14.0
 ---> Using cache
 ---> db01143e2e13
Step 4/15 : ARG NEXUS_BUILD=01
 ---> Using cache
 ---> 0a70ee3b3328
Step 5/15 : COPY . /nexus-repository-conan/
 ---> Using cache
 ---> 74c69585b61c
Step 6/15 : RUN cd /nexus-repository-conan/; sed -i "s/3.14.0-04/${NEXUS_VERSION}-${NEXUS_BUILD}/g" pom.xml;     mvn clean package;
 ---> Running in c945242673b4
[INFO] Scanning for projects...
Downloading from central: https://repo.maven.apache.org/maven2/org/sonatype/nexus/plugins/nexus-plugins/3.14.0-01/nexus-plugins-3.14.0-01.pom
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[FATAL] Non-resolvable parent POM for org.sonatype.repository:nexus-repository-conan:0.0.5: Could not find artifact org.sonatype.nexus.plugins:nexus-plugins:pom:3.14.0-01 in central (https://repo.maven.apache.org/maven2) and 'parent.relativePath' points at wrong local POM @ line 19, column 11
 @
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]
[ERROR]   The project org.sonatype.repository:nexus-repository-conan:0.0.5 (/nexus-repository-conan/pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for org.sonatype.repository:nexus-repository-conan:0.0.5: Could not find artifact org.sonatype.nexus.plugins:nexus-plugins:pom:3.14.0-01 in central (https://repo.maven.apache.org/maven2) and 'parent.relativePath' points at wrong local POM @ line 19, column 11 -> [Help 2]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/UnresolvableModelException
The command '/bin/sh -c cd /nexus-repository-conan/; sed -i "s/3.14.0-04/${NEXUS_VERSION}-${NEXUS_BUILD}/g" pom.xml;     mvn clean package;' returned a non-zero code: 1```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,21 +1,21 @@
-ARG NEXUS_VERSION=3.14.0
+ARG NEXUS_VERSION=3.15.1
 
 FROM maven:3-jdk-8-alpine AS build
-ARG NEXUS_VERSION=3.14.0
+ARG NEXUS_VERSION=3.15.1
 ARG NEXUS_BUILD=01
 
 COPY . /nexus-repository-conan/
-RUN cd /nexus-repository-conan/; sed -i "s/3.14.0-04/${NEXUS_VERSION}-${NEXUS_BUILD}/g" pom.xml; \
+RUN cd /nexus-repository-conan/; sed -i "s/3.15.1-01/${NEXUS_VERSION}-${NEXUS_BUILD}/g" pom.xml; \
     mvn clean package;
 
 FROM sonatype/nexus3:$NEXUS_VERSION
-ARG NEXUS_VERSION=3.14.0
+ARG NEXUS_VERSION=3.15.1
 ARG NEXUS_BUILD=01
-ARG CONAN_VERSION=0.0.5
+ARG CONAN_VERSION=0.0.6
 ARG TARGET_DIR=/opt/sonatype/nexus/system/org/sonatype/nexus/plugins/nexus-repository-conan/${CONAN_VERSION}/
 USER root
 RUN mkdir -p ${TARGET_DIR}; \
-    sed -i 's@nexus-repository-maven</feature>@nexus-repository-maven</feature>\n        <feature prerequisite="false" dependency="false" version="0.0.5">nexus-repository-conan</feature>@g' /opt/sonatype/nexus/system/org/sonatype/nexus/assemblies/nexus-core-feature/${NEXUS_VERSION}-${NEXUS_BUILD}/nexus-core-feature-${NEXUS_VERSION}-${NEXUS_BUILD}-features.xml; \
-    sed -i 's@<feature name="nexus-repository-maven"@<feature name="nexus-repository-conan" description="org.sonatype.nexus.plugins:nexus-repository-conan" version="0.0.5">\n        <details>org.sonatype.nexus.plugins:nexus-repository-conan</details>\n        <bundle>mvn:org.sonatype.nexus.plugins/nexus-repository-conan/0.0.4</bundle>\n    </feature>\n    <feature name="nexus-repository-maven"@g' /opt/sonatype/nexus/system/org/sonatype/nexus/assemblies/nexus-core-feature/${NEXUS_VERSION}-${NEXUS_BUILD}/nexus-core-feature-${NEXUS_VERSION}-${NEXUS_BUILD}-features.xml;
+    sed -i 's@nexus-repository-maven</feature>@nexus-repository-maven</feature>\n        <feature prerequisite="false" dependency="false" version="0.0.6">nexus-repository-conan</feature>@g' /opt/sonatype/nexus/system/org/sonatype/nexus/assemblies/nexus-core-feature/${NEXUS_VERSION}-${NEXUS_BUILD}/nexus-core-feature-${NEXUS_VERSION}-${NEXUS_BUILD}-features.xml; \
+    sed -i 's@<feature name="nexus-repository-maven"@<feature name="nexus-repository-conan" description="org.sonatype.nexus.plugins:nexus-repository-conan" version="0.0.6">\n        <details>org.sonatype.nexus.plugins:nexus-repository-conan</details>\n        <bundle>mvn:org.sonatype.nexus.plugins/nexus-repository-conan/0.0.6</bundle>\n    </feature>\n    <feature name="nexus-repository-maven"@g' /opt/sonatype/nexus/system/org/sonatype/nexus/assemblies/nexus-core-feature/${NEXUS_VERSION}-${NEXUS_BUILD}/nexus-core-feature-${NEXUS_VERSION}-${NEXUS_BUILD}-features.xml;
 COPY --from=build /nexus-repository-conan/target/nexus-repository-conan-${CONAN_VERSION}.jar ${TARGET_DIR}
 USER nexus

--- a/README.md
+++ b/README.md
@@ -58,11 +58,11 @@ If everything checks out, the bundle for Conan should be available in the `targe
 
 #### Build with Docker
 
-`docker build -t nexus-repository-conan:0.0.5 .`
+`docker build -t nexus-repository-conan:0.0.6 .`
 
 #### Run as a Docker container
 
-`docker run -d -p 8081:8081 --name nexus nexus-repository-conan:0.0.5` 
+`docker run -d -p 8081:8081 --name nexus nexus-repository-conan:0.0.6` 
 
 For further information like how to persist volumes check out [the GitHub repo for our official image](https://github.com/sonatype/docker-nexus3).
 
@@ -91,7 +91,7 @@ good installation path if you are just testing or doing development on the plugi
   # sudo su - nexus
   $ cd <nexus_dir>/bin
   $ ./nexus run
-  > bundle:install file:///tmp/nexus-repository-conan-0.0.5.jar
+  > bundle:install file:///tmp/nexus-repository-conan-0.0.6.jar
   > bundle:list
   ```
   (look for org.sonatype.nexus.plugins:nexus-repository-conan ID, should be the last one)
@@ -103,7 +103,7 @@ good installation path if you are just testing or doing development on the plugi
 
 For more permanent installs of the nexus-repository-conan plugin, follow these instructions:
 
-* Copy the bundle (nexus-repository-conan-0.0.5.jar) into <nexus_dir>/deploy
+* Copy the bundle (nexus-repository-conan-0.0.6.jar) into <nexus_dir>/deploy
 
 This will cause the plugin to be loaded with each restart of Nexus Repository. As well, this folder is monitored
 by Nexus Repository and the plugin should load within 60 seconds of being copied there if Nexus Repository
@@ -113,20 +113,20 @@ is running. You will still need to start the bundle using the karaf commands men
 
 If you are trying to use the Conan plugin permanently, it likely makes more sense to do the following:
 
-* Copy the bundle into `<nexus_dir>/system/org/sonatype/nexus/plugins/nexus-repository-conan/0.0.5/nexus-repository-conan-0.0.5.jar`
+* Copy the bundle into `<nexus_dir>/system/org/sonatype/nexus/plugins/nexus-repository-conan/0.0.6/nexus-repository-conan-0.0.6.jar`
 * If you are using OSS edition, make these mods in: `<nexus_dir>/system/com/sonatype/nexus/assemblies/nexus-oss-feature/3.x.y/nexus-oss-feature-3.x.y-features.xml`
 * If you are using PRO edition, make these mods in: `<nexus_dir>/system/com/sonatype/nexus/assemblies/nexus-pro-feature/3.x.y/nexus-pro-feature-3.x.y-features.xml`
    ```
          <feature version="3.x.y.xy" prerequisite="false" dependency="false">nexus-repository-rubygems</feature>
-         <feature version="0.0.5" prerequisite="false" dependency="false">nexus-repository-conan</feature>
+         <feature version="0.0.6" prerequisite="false" dependency="false">nexus-repository-conan</feature>
          <feature version="3.x.y.xy" prerequisite="false" dependency="false">nexus-repository-yum</feature>
      </features>
    ```
    And
    ```
-       <feature name="nexus-repository-conan" description="org.sonatype.nexus.plugins:nexus-repository-conan" version="0.0.5">
+       <feature name="nexus-repository-conan" description="org.sonatype.nexus.plugins:nexus-repository-conan" version="0.0.6">
         <details>org.sonatype.nexus.plugins:nexus-repository-conan</details>
-        <bundle>mvn:org.sonatype.nexus.plugins/nexus-repository-conan/0.0.5</bundle>
+        <bundle>mvn:org.sonatype.nexus.plugins/nexus-repository-conan/0.0.6</bundle>
        </feature>
     </features>
    ```

--- a/pom.xml
+++ b/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <groupId>org.sonatype.nexus.plugins</groupId>
     <artifactId>nexus-plugins</artifactId>
-    <version>3.14.0-04</version>
+    <version>3.15.0-01</version>
   </parent>
   <groupId>org.sonatype.repository</groupId>
   <artifactId>nexus-repository-conan</artifactId>
   <name>${project.groupId}:${project.artifactId}</name>
-  <version>0.0.5</version>
+  <version>0.0.6</version>
   <inceptionYear>2017</inceptionYear>
   <packaging>bundle</packaging>
 

